### PR TITLE
Please check mobile view

### PR DESCRIPTION
**Pull request by abhirupa-tech**
https://developerabhirupa.github.io/QuizIEEEtask/

### CODE REVIEW

File: users/DeveloperAbhirupa/Task3/index.html [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+<head>
+<title>GameON</title>
+<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-alpha.6/css/bootstrap.min.css" integrity="sha384-rwoIResjU2yc3z8GV/NPeZWAv56rSmLldC3R/AZzGRnGxQQKnKkoFVhFQhNUwEyJ" crossorigin="anonymous">
+<script src="https://code.jquery.com/jquery-3.1.1.slim.min.js" integrity="sha384-A7FZj7v+d/sdmMqp/nOQwliLvUsJfDHW+k9Omg/a/EheAdgtzNs3hpfag6Ed950n" crossorigin="anonymous"></script>
+<script src="https://cdnjs.cloudflare.com/ajax/libs/tether/1.4.0/js/tether.min.js" integrity="sha384-DztdAPBWPRXSA/3eYEEUWrWCy7G5KFbe8fFjk5JAIxUYHKkDx6Qin1DkWx51bBrb" crossorigin="anonymous"></script>
+<script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-alpha.6/js/bootstrap.min.js" integrity="sha384-vBWWzlZJ8ea9aCX4pEW3rVHjgjt7zpkNpZk+02D9phzyeVkE+jo0ieGizqPLForn" crossorigin="anonymous"></script>
+<link href="https://fonts.googleapis.com/css?family=Comfortaa|Great+Vibes" rel="stylesheet">
+<link href="https://fonts.googleapis.com/css?family=Titillium+Web:900" rel="stylesheet">
+<script type="text/javascript" src="particles.js"></script>
+<link rel="stylesheet" href="stylesheet1.css">
```
Comment (armudgal) [inline users/DeveloperAbhirupa/Task3/index.html:11]: You have to write your own `css`. Please make the changes

File: users/DeveloperAbhirupa/Task3/index2.html [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+<title>Quiz</title>
+<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-alpha.6/css/bootstrap.min.css" integrity="sha384-rwoIResjU2yc3z8GV/NPeZWAv56rSmLldC3R/AZzGRnGxQQKnKkoFVhFQhNUwEyJ" crossorigin="anonymous">
+<script src="https://code.jquery.com/jquery-3.1.1.slim.min.js" integrity="sha384-A7FZj7v+d/sdmMqp/nOQwliLvUsJfDHW+k9Omg/a/EheAdgtzNs3hpfag6Ed950n" crossorigin="anonymous"></script>
+<script src="https://cdnjs.cloudflare.com/ajax/libs/tether/1.4.0/js/tether.min.js" integrity="sha384-DztdAPBWPRXSA/3eYEEUWrWCy7G5KFbe8fFjk5JAIxUYHKkDx6Qin1DkWx51bBrb" crossorigin="anonymous"></script>
+<script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-alpha.6/js/bootstrap.min.js" integrity="sha384-vBWWzlZJ8ea9aCX4pEW3rVHjgjt7zpkNpZk+02D9phzyeVkE+jo0ieGizqPLForn" crossorigin="anonymous"></script>
+<link href="https://fonts.googleapis.com/css?family=Amaranth" rel="stylesheet">
+<link href="https://fonts.googleapis.com/css?family=Titillium+Web:900" rel="stylesheet">
+<script type="text/javascript" src="particles.js"></script>
+<script type="text/javascript" src="script1.js"></script>
+<link rel="stylesheet" href="stylesheet1.css">
```
Comment (armudgal) [inline users/DeveloperAbhirupa/Task3/index2.html]: External css

**Review by armudgal — CHANGES_REQUESTED** [review]
Hi, good work on the additions. I would suggest focus on the basics first, looking forward to your updated PR soon.

File: users/DeveloperAbhirupa/Task3/index2.html [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+
+
+
+<body background="img/pic1.jpg">
+  <div class="container C1" >
+  1.  What was the first commercial computer that had a GUI and a mouse?
+
+  <form id="form">
+      <label><input type="radio"  name="optradio1" > PARAM</input></label><br>
+      <label><input type="radio"  name="optradio1" id="1"> Apple Lisa</label><br>
```
Comment (armudgal) [inline users/DeveloperAbhirupa/Task3/index2.html]: `id`s are giving away the answer. There is a better implementation to this ;)

File: users/DeveloperAbhirupa/Task3/index2.html [hunk 4]
```diff
@@ -0,0 +1,136 @@
...
+
+  <form id="form">
+      <label><input type="radio"  name="optradio10" > Its 700 year old bamboo tree</input></label><br>
+      <label><input type="radio"  name="optradio10" > World's most silent temple</label><br>
+      <label><input type="radio" name="optradio10" > Cleanest Temple</label><br>
+      <label><input type="radio" name="optradio10" id="10" > 20,000 rats as residents</label>
+  </form>
+  </div>
+
+<div class="cb" id="s">Click Submit when you finish.</div>
```
Comment (armudgal) [inline users/DeveloperAbhirupa/Task3/index2.html]: Try for better naming conventions

File: users/DeveloperAbhirupa/Task3/index2.html [hunk 5]
```diff
@@ -0,0 +1,136 @@
...
+<br><br>
+  <div class="b1" align="middle"><button class="btn sub" align="ceb" onclick="updateScore()"> SUBMIT</button> </div>
+
+<br><br><br>
```
Comment (armudgal) [inline users/DeveloperAbhirupa/Task3/index2.html]: Nope, use `margin`

File: users/DeveloperAbhirupa/Task3/script1.js [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+    user[0] = document.getElementById("1").checked;
+    user[1] = document.getElementById("2").checked;
+    user[2] = document.getElementById("3").checked;
+    user[3] = document.getElementById("4").checked;
+    user[4] = document.getElementById("5").checked;
+    user[5] = document.getElementById("6").checked;
+    user[6] = document.getElementById("7").checked;
+    user[7] = document.getElementById("8").checked;
+    user[8] = document.getElementById("9").checked;
+    user[9] = document.getElementById("10").checked;
```
Comment (armudgal) [inline users/DeveloperAbhirupa/Task3/script1.js]: This is a very very bad way to do this.

File: users/DeveloperAbhirupa/Task3/stylesheet1.css [hunk 7]
```diff
@@ -0,0 +1,123 @@
...
+  transition: 1s;
+}
+.ca,.cb{
+  font-family: 'Amaranth', sans-serif;
+  font-family: 'Titillium Web', sans-serif;
+  font-size: 50px;
+  text-align: center;
+}
+
+@media all and (max-width: 500px)
```
Comment (armudgal) [inline users/DeveloperAbhirupa/Task3/stylesheet1.css]: I don't like `media-queries`. There is `bootstrap` to your rescue.

File: users/DeveloperAbhirupa/Task3/index.html (hunk 1 above)
Comment (abhirupa-tech) [reply users/DeveloperAbhirupa/Task3/index.html:11]: @kira0204 I have my own external css: "stylesheet1.css"

**Review by armudgal — CHANGES_REQUESTED** [review]

File: users/DeveloperAbhirupa/Task3/index.html (hunk 1 above)
Comment (armudgal) [reply users/DeveloperAbhirupa/Task3/index.html:11]: It has media queries, please remove them then

**Comment by abhirupa-tech** [comment]
JS altered, mobile view improved.
Answers not revealed in the source file.
JQuery implemented.

I want my quiz to automatically end, when the user navigates out of the active tab, please drop a hint ;)

File: users/DeveloperAbhirupa/Task3/stylesheet1.css [hunk 8]
```diff
@@ -91,33 +91,37 @@ background: linear-gradient(white,grey);
   text-align: center;
 }
 
-@media all and (max-width: 500px)
+@media only screen
```
Comment (armudgal) [inline users/DeveloperAbhirupa/Task3/stylesheet1.css]: Again, you could have done this all via `bootstrap` ;)

File: users/DeveloperAbhirupa/Task3/script1.js [hunk 9]
```diff
@@ -7,28 +7,18 @@ var score=0;
...
-    user[2] = document.getElementById("3").checked;
-    user[3] = document.getElementById("4").checked;
-    user[4] = document.getElementById("5").checked;
-    user[5] = document.getElementById("6").checked;
-    user[6] = document.getElementById("7").checked;
-    user[7] = document.getElementById("8").checked;
-    user[8] = document.getElementById("9").checked;
-    user[9] = document.getElementById("10").checked;
-
+    user=['1','4','11','14','18', '21','24','29','32','39'];
```
Comment (armudgal) [inline users/DeveloperAbhirupa/Task3/script1.js]: Great :)

**Review by armudgal — APPROVED** [review]

**Pull request merged**